ompi/mpiext/shortfloat/README.txt

```diff
@@ -0,0 +1,17 @@
+Copyright (c) 2018      FUJITSU LIMITED.  All rights reserved.
+
+$COPYRIGHT$
+
+This extension provides additional MPI datatypes MPIX_SHORT_FLOAT,
+MPIX_C_SHORT_FLOAT_COMPLEX, and MPIX_CXX_SHORT_FLOAT_COMPLEX, which
+are proposed with the MPI_ prefix in the MPI Forum as of Dec. 2018.
```

Comment:
Similar to before: I'd put a reference to the MPI v4.0 specification here, just to give readers a little context (e.g., if someone reads this years from now, after MPI v5.0 already exists... etc.).